core/src/outputs.rs

```diff
@@ -1,6 +1,5 @@
-use crate::{stack::STACK_TOP_SIZE, StackTopState};
-
 use super::{Felt, StarkField};
+use crate::{stack::STACK_TOP_SIZE, StackTopState};
 use winter_utils::collections::Vec;
 
 // PROGRAM OUTPUTS
```

Comment:
nit: Should probably be changed to STACK OUTPUTS.
Similar change should probably be made on line 10.

Reply:
Fixed!

Reply:
Seems like this might have been reverted.

Reply:
@vlopes11 confirmed that he will have those changes in the subsequent PR.